.github/workflows/assign-reviewer.yml

```diff
@@ -0,0 +1,12 @@
+name: "Auto Assign"
+on:
+  - pull_request
```

Comment:
Not sure if this token can even assign reviewers.
@zowoq I guess I need pull_request_target here?

Reply:
https://github.com/shufo/auto-assign-reviewer-by-files#use-of-team-reviewers-results-in-could-not-resolve-to-a-node-with-the-global-id-of-error

I think `pull_request` should work, only looks like it's problematic with teams. As it's `pull_request` should be able to test it in this PR?

Like the rest of the actions in this repo `pull_request` won't run automatically for new contributors so CODEOWNERS may be a better option here if you want consistent notifications.

Reply:
> `pull_request` won't run automatically for new contributors

What exactly does this mean? Will it not run if a new contributor makes a pull request? Why?

Reply:
https://github.blog/changelog/2021-04-22-github-actions-maintainers-must-approve-first-time-contributor-workflow-runs/

Reply:
> What exactly does this mean? Will it not run if a new contributor makes a pull request? Why?

Github action CI are skipped for first-time contributors.

Reply:
> https://github.com/shufo/auto-assign-reviewer-by-files#use-of-team-reviewers-results-in-could-not-resolve-to-a-node-with-the-global-id-of-error
> 
> I think `pull_request` should work, only looks like it's problematic with teams. As it's `pull_request` should be able to test it in this PR?
> 
> Like the rest of the actions in this repo `pull_request` won't run automatically for new contributors so CODEOWNERS may be a better option here if you want consistent notifications.

Yes but CODEOWNERS requires write access to the repo, which @fricklerhandwerk does not have. So this is the second best solution. At least once someone allows CI to be run for a PR than it will assign him.

Reply:
In my case I do have merge access, but we want more maintainers for documentation, who can review changes and want to be notified for that purpose. But they don't necessarily have to merge them themselves.

It may be fine to have this as an intermediate solution if someone wants click on the button for every PR. Not sure if I can and want to afford the time.

@edolstra what do you think about giving the current docs team lead merge access and make it their responsibility? @LucPerkins would it be okay for you to take this on?

Reply:
Sorry, looks like this did need to be `pull_request_target`. 

https://github.com/NixOS/nix/pull/7398